### DISCUSSION TITLE
When I open the web version and play Online:  https://nzp.gay/ when I open the console there is no ip code

### DISCUSSION THREAD

**Discussion by MrCrowTheBird** [nzp-team/nzportable#505] [General]
![image](https://github.com/nzp-team/nzportable/assets/152709130/f7fa79d5-957b-4b0e-bdae-2cedeb7daa03)

**Comment by MotoLegacy** [comment]
`Listening on tls://master.frag-net.com:27950/NZP-REBOOT/2587` <---- `/2587` is your "Room ID"

**Reply by MrCrowTheBird** [reply]
Thanks it works now :D
